src/graphql/main/schema.graphql

```diff
@@ -387,6 +388,11 @@ enum InvoicePaymentStatus {
   PENDING
 }
 
+type JwtPayload {
```

Comment:
do we need a new type here? shouldn't this be `AuthTokenPayload` ?

Reply:
the jwt is a different length, so the AuthToken validator fails here https://github.com/GaloyMoney/galoy/blob/main/src/graphql/types/scalar/auth-token.ts#L17